test/IntegrationTests/ContinuousProfilerContextTrackingTests.cs

```diff
@@ -45,19 +42,41 @@ private bool AssertAllProfiles(ICollection<ExportProfilesServiceRequest> profile
             var samplesInBatch = profile.Sample;
 
             var samplesWithTraceContext = samplesInBatch.Where(s => s.HasLinkIndex).ToList();
-
+#if NET
             Assert.True(samplesWithTraceContext.Count <= 1, "at most one sample in a batch should have trace context associated.");
-
+#endif
             totalSamplesWithTraceContextCount += samplesWithTraceContext.Count;
             if (samplesWithTraceContext.FirstOrDefault() is { } sampleWithTraceContext)
             {
-                managedThreadsWithTraceContext.Add(profile.AttributeTable[sampleWithTraceContext.AttributeIndices.Single()].Value.StringValue);
+                var threadId = GetThreadName(profile, sampleWithTraceContext);
+                Output.WriteLine($"Found trace context on thread: {threadId}");
+                managedThreadsWithTraceContext.Add(threadId!);
```

Comment:
I have reverted this file, except removing conditional compilation at the beginning of this file.
It still passes.
What is the reason behind changes?